gpu-operator/getting-started.rst

```diff
@@ -173,6 +173,17 @@ To view all the options, run ``helm show values nvidia/gpu-operator``.
        Set this value to ``false`` when using the Operator on systems with pre-installed drivers.
      - ``true``
 
+   * - ``kernelModuleType``
+     - Specifies the type of the NVIDIA GPU Kernel modules to use.
+       Valid values are ``auto`` (default), ``proprietary``, and ``open``. 
+       
+       ``Auto`` means that the recommended kernel module type (open or proprietary) is chosen based on the GPU devices on the host and the driver branch used.
+       Note, ``auto`` is only supported with the 570.86.15 and 570.124.06 or later driver containers. 
+       550 and 535 branch drivers do not yet support this mode.
+       ``Open`` means the open kernel module is used.
+       ``Proprietary`` means the proprietary module is used.
+     - ``auto``
```

Comment:
Should this line be removed?

Reply:
this adds "auto" to the last column on the table as the default option. so it should be here to complete the table.

Reply:
Understoof